en/components/autocomplete.md

```diff
@@ -0,0 +1,254 @@
+---
+title: Autocomplete Directive
+_description: The igxAutocomplete directive provides a way to enhance a text input by showing a panel of suggested options provided by the developer.
+_keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI widgets, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components Library, Angular Autocomplete components, Angular Autocomplete directives, Angular Autocomplete controls
+---
+
+## Autocomplete
+<p class="highlight">
+
+The [**igxAutocomplete**]({environment:angularApiUrl}/classes/igxautocompletedirective.html) directive provides a way to enhance a text input by showing a [drop-down]({environment:angularApiUrl}/classes/igxdropdowncomponent.html) with suggested options that are provided by the developer.
+
+The [igx-autocomplete]({environment:angularApiUrl}/classes/igxautocompletedirective.html) suggestions will show once you start typing in the text input. By default, the first item is always highlighted, so that it can be easily selected using the `Enter` key. An other way of selecting an item is to click on it from the list, then the input value will be automatically updated. Also, the drop-down will close. When an item is selected, the `onItemSelected` event is fired, if this event is canceled, the item will not be selected and the drop-down will not close.
```

Comment:
so that it can be easily selected using the `Enter` key -> which provides an easy way to select it using the 'Enter' key
An other way of selecting an item is to click on it from the list, then the input value will be automatically updated. -> Clicking on the item from the select will select it too, and the input value will be automatically updated, which will result closing of the dropdown